docker-compose.yml

```diff
@@ -22,6 +22,8 @@ services:
       - nginx
     environment:
       - DOCUMENTSERVER=http://documentserver
-    command: ["ruby", "./helpers/wait_for_documentserver_start.rb"]
+    command: >
+      bash -c "ruby helpers/wait_for_documentserver_start.rb
```

Comment:
Maybe place both commands to entrypoint.sh like [this](https://github.com/ONLYOFFICE/testing-wrata/blob/master/entrypoint.sh)

This allow more clear calls in docker-compose